src/server.js

```diff
@@ -15,11 +15,8 @@ app.listen(port, () => {
 });
 
 const bot = new Telegraf(process.env.BOT_TOKEN);
-bot.start((ctx) => ctx.reply("Welcome"));
-bot.help((ctx) => ctx.reply("Send me a sticker"));
-bot.on("sticker", (ctx) => ctx.reply("👍"));
-bot.hears("hi", (ctx) => ctx.reply("Hey there"));
-bot.launch();
+import { trackerController as controller } from "./controllers/tracker.controller.js";
```

Comment:
1. Move import to the top of the file

2. Thoughts on naming this import as a `tracker` as opposed to a `controller`? 
that way if we have multiple controller, we'd avoid a name clash

Reply:
good catch(es). thanks!